src/utils/FormLabel.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+interface FormLabelProps{
+    labelName:string,
+    id:string,
+    classes:string
```

Comment:
Its better to have default class, so that you wont have to repeat for other input element label
for those which has customized label style then you can pass styles in classNames.